crates/wash-lib/src/parser/mod.rs

```diff
@@ -305,19 +305,19 @@ pub enum WasmTarget {
         alias = "wasm32-wasi-preview1",
         alias = "wasm32-wasip1"
     )]
-    WasiPreview1,
-    #[serde(alias = "wasm32-wasi-preview2", alias = "wasm32-wasip2")]
-    WasiPreview2,
+    WasiP1,
+    #[serde(alias = "wasm32-wasip2", alias = "wasm32-wasip2")]
+    WasiP2,
```

Comment:
```suggestion
    WasiP1,
    #[serde(alias = "wasm32-wasip2", alias = "wasm32-preview2", alias = "wasm32-wasip2")]
    WasiP2,
```

Make sure to include an alias for the old type for WasiP1 and WasiP2 to keep this backwards compatible with wasmcloud.tomls

Reply:
ack last minute replace all. TY!